go-runtime/ftl/database.go

```diff
@@ -29,10 +31,17 @@ func PostgresDatabase(name string) Database {
 			if err != nil {
 				return nil, fmt.Errorf("failed to get database %q: %w", name, err)
 			}
-			db, err := sql.Open("pgx", dsn)
+			db, err := otelsql.Open("pgx", dsn)
 			if err != nil {
 				return nil, fmt.Errorf("failed to open database %q: %w", name, err)
 			}
+
+			// sets db.system and db.name attributes
+			metricAttrs := otelsql.WithAttributes(semconv.DBSystemPostgreSQL, semconv.DBNameKey.String(name))
```

Comment:
Let's also add `ftl.is_user_service` and `ftl.sql.module.name` as attributes?
```suggestion
			metricAttrs := otelsql.WithAttributes(
				semconv.DBSystemPostgreSQL,
				semconv.DBNameKey.String(name),
				attribute.Bool("ftl.is_user_service", true),
				attribute.String("ftl.sql.module.name", provider.module),
			)
```